libs/application/template-api-modules/src/lib/modules/templates/signature-collection/parliamentary-list-signing/parliamentary-list-signing.service.ts

```diff
@@ -0,0 +1,104 @@
+import { Injectable } from '@nestjs/common'
+import { TemplateApiModuleActionProps } from '../../../../types'
+import { BaseTemplateApiService } from '../../../base-template-api.service'
+import { ApplicationTypes } from '@island.is/application/types'
+import {
+  ReasonKey,
+  SignatureCollectionClientService,
+} from '@island.is/clients/signature-collection'
+import { TemplateApiError } from '@island.is/nest/problem'
+import { errorMessages } from '@island.is/application/templates/signature-collection/presidential-list-signing'
+import { ProviderErrorReason } from '@island.is/shared/problem'
+
+@Injectable()
+export class ParliamentaryListSigningService extends BaseTemplateApiService {
+  constructor(
+    private signatureCollectionClientService: SignatureCollectionClientService,
+  ) {
+    super(ApplicationTypes.PARLIAMENTARY_LIST_SIGNING)
+  }
+
+  async signList({ auth, application }: TemplateApiModuleActionProps) {
+    const listId = application.answers.listId
+      ? (application.answers.listId as string)
+      : (application.externalData.getList.data as any)[0].id
```

Comment:
Would be nicer to type this methinks.
You could probably add the following line
> `export * from './lib/types/list.dto'`
to the file `libs/clients/signature-collection/src/index.ts`

and then import it here with

`import { List } from '@island.is/clients/signature-collection'`

and then finally do

`: (application.externalData.getList.data as List[])[0].id`